examples/editor/index.js

```diff
@@ -255,6 +263,8 @@ const superRefresh = (opts) => {
     const vizStr = document.getElementById('styleEntry').value;
     const viz = new carto.Viz(vizStr);
 
+    saveConfig();
```

Comment:
Why is this added?

Reply:
I added it because currently, every time there is an error, if you change the fields after that those changes are not saved. There may be a better way to do so, actually. Any suggestion?

Reply:
OK. I would move the function to the top to save after Source or Viz errors. What do you think?

Reply:
👍 Agree!